crates/store/src/lib.rs

```diff
@@ -1,11 +1,11 @@
 use std::time::Duration;
 
 mod blocks;
-mod db;
+pub mod db;
```

Comment:
Why does `db` need to be `pub`?

Reply:
I got it out, but it's giving me a problem now that the `State` module is public, the documentation of one of its methods (`get_notes_by_id`) fails when running `make doc`, since it refers to `NoteRecord`, and it's from the `Db` module.

Not really sure which path take here, the crate doesn't have features so I don't want to create one, but moving the public visibility of the `State` under a feature fixes the issue. Making Db public also fixes it.

Reply:
I think we can just modify the doc comment not to link to `NoteRecord`.